core/shared/src/main/scala/org/http4s/util/io/package.scala

```diff
@@ -0,0 +1,91 @@
+/** TODO fs2 port
```

Comment:
I think we can drop this entire file.

Reply:
This one is removed on master. Maybe a rebase is needed?